mrtt-ui/src/components/SocioeconomicAndGovernance/SocioeconomicAndGovernanceStatusAndOutcomesForm.js

```diff
@@ -0,0 +1,233 @@
+        </PageTitle>
+        <PageSubtitle>{site_name}</PageSubtitle>
+      </FormPageHeader>
+      <QuestionNav
+        isFormSaving={isSubmitting}
+        isFormSaveError={isSubmitError}
+        onFormSave={validateInputs(handleSubmit)}
+        currentSection='socioeconomic-and-governance-status'
+      />
+      <FormValidationMessageIfErrors formErrors={errors} />
+
+      <Form>
+        <FormQuestionDiv>
+          <StickyFormLabel>{questions.dateOfOutcomesAssessment.question}</StickyFormLabel>
+          <Controller
+            name='dateOfOutcomesAssessment'
+            control={control}
+            defaultValue={null}
+            render={({ field }) => (
+              <LocalizationProvider dateAdapter={AdapterDateFns} {...field} ref={null}>
+                <Stack spacing={3}>
+                  <MobileDatePicker
+                    id='date-of-outcomes-assessment'
+                    label='date'
+                    value={field.value}
+                    onChange={(newValue) => {
+                      field.onChange(newValue?.toISOString())
+                    }}
+                    renderInput={(params) => <TextField {...params} />}
+                  />
+                </Stack>
+              </LocalizationProvider>
+            )}
+          />
+          <ErrorText>{errors.dateOfOutcomesAssessment?.message}</ErrorText>
+        </FormQuestionDiv>
+        <FormQuestionDiv>
+          <StickyFormLabel>{questions.changeInGovernance.question}</StickyFormLabel>
+          <Controller
+            name='changeInGovernance'
+            control={control}
+            defaultValue=''
+            render={({ field }) => (
+              <TextField {...field} select value={field.value} label='select'>
+                {questions.changeInGovernance.options.map((item, index) => (
+                  <MenuItem key={index} value={item}>
+                    {item}
+                  </MenuItem>
+                ))}
+              </TextField>
+            )}
+          />
+          <ErrorText>{errors.changeInGovernance?.message}</ErrorText>
+        </FormQuestionDiv>
+        {changeInGovernanceWatcher === 'Yes' ? (
+          <FormQuestionDiv>
+            <CheckboxGroupWithLabelAndController
+              fieldName='currentGovenance'
+              reactHookFormInstance={reactHookFormInstance}
+              options={questions.currentGovenance.options}
+              question={questions.currentGovenance.question}
+              shouldAddOtherOptionWithClarification={false}
+            />
+            <ErrorText>{errors.currentGovenance?.selectedValues?.message}</ErrorText>
+          </FormQuestionDiv>
+        ) : null}
+        <FormQuestionDiv>
+          <StickyFormLabel>{questions.changeInTenureArrangement.question}</StickyFormLabel>
+          <Controller
+            name='changeInTenureArrangement'
+            control={control}
+            defaultValue=''
```

Comment:
Nit: If this still works when you dont have the inline defaultValue (as in not a bug fix to MUI), I think its better to be consistent with where we declare defaults and have them in the defaultValues object.

Reply:
agreed that its good to be consistent. The controller often bugs out without a defaultValue though